MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,489 @@
+</td>
+<td>
+{{'[Entry]'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+MultilineText
+</td>
+<td>
+{{'[DataFormMultiLineTextEditor]'| markdownify }}
+</td>
+<td>
+The String type property with multi line text.
+[DataType(DataType.MultilineText)] 
+</td>
+<td>
+{{'[Editor]'| markdownify }}
+</td>
+</tr>
+</tr>
+<tr>
+<td>
+Password
+</td>
+<td>
+{{'[DataFormPasswordEditor]'| markdownify }}
+</td>
+<td>
+The String type property and property with 
+[DataType(DataType.Password)] attribute.
+</td>
+<td>
+{{'[Entry]'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Checkbox
+</td>
+<td>
+{{'[DataFormCheckBoxEditor]'| markdownify }}
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[CheckBox]'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Switch
+</td>
+<td>
+{{'[DataFormSwitchEditor]'| markdownify }}
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[Switch]'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Date Picker
+</td>
+<td>
+{{'[DataFormDatePickerEditor]'| markdownify }}
+</td>
+<td>
+The DateTime type property and the property with [DataType(DataType.Date)] and [DataType(DataType.DateTime)] attributes.
+</td>
+<td>
+{{'[DatePicker]| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Time Picker
+</td>
+<td>
+{{'[DataFormTimePickerEditor]'| markdownify }}
+</td>
+<td>
+The property with [DataType(DataType.Time)] attribute.
+</td>
+<td>
+{{'[TimePicker]'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Picker
+</td>
+<td>
+{{'[DataFormPickerEditor]'| markdownify }}
+</td>
+<td>
+Enum and List type property. 
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[Picker]'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+{{'[DataFormComboBoxEditor]'| markdownify }}
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfComboBox]'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+{{'[DataFormAutoCompleteEditor]'| markdownify }}
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfAutocomplete]'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+{{'[DataFormRadioGroupEditor]'| markdownify }}
+</td>
+<td>
+Enum and List type property. 
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfRadioGroup]'| markdownify }}
+</td>
+</tr>
+</table>
+
+## Changing editor for type
+
+By default, the editors will be loaded based on the previous table. To change the editor for any type, use the [RegisterEditor] method and specify the type and editor.
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.RegisterEditor(typeof(string), DataFormEditorType.MultilineText);
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the `MultilineText` editor will be loaded for the string type instead of `Text` editor.
+
+## Changing editor for property
+
+To change the editor for any property, use the [RegisterEditor] method and specify the property name and editor.
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.RegisterEditor(nameof(On), DataFormEditorType.Switch);
```

Comment:
if we use nameof then should use model class like nameof(DataModel.On) => change it using model class or make on as a string value "On"